minet/instagram/formatters.py

```diff
@@ -49,16 +58,26 @@
 )
 
 
+def get_usertags(item, usertags):
```

Comment:
Change the name of this function to reflect mutation or change the behavior of the function to return the currently found usertags and then aggregate them where you use the function.